crates/wdl-engine/src/config.rs

```diff
@@ -1210,6 +1211,507 @@ impl TesBackendConfig {
     }
 }
 
+/// Configuration for the Apptainer container runtime.
```

Comment:
Knowing what I know right now, I would have a slight preference to keep these in their respective modules close to the implementation. That being said, I can see a number of reasons why one might want to centralize them here, so I'll just ask: what was the reasoning for moving these?

Reply:
The `config` module is public whereas their previous modules are no longer public; it also consolidates all the types exposed via `Config`.